configure.ac

```diff
@@ -585,7 +585,10 @@ AC_PATH_PROG([XSLTPROC], [xsltproc])
 AC_PATH_PROG([XMLCATALOG], [xmlcatalog])
 dnl BASH is already an environment variable, so use something else
 AC_PATH_PROG([BASH_PATH], [bash])
+
 PKG_PROG_PKG_CONFIG
+PKG_NOARCH_INSTALLDIR
```

Comment:
This isn't available until pkg-config 0.27 (2012-07-11). We currently (implicitly) require just 0.16, and we loosely target about 10 years back for a build environment. Let's use ${datadir}/pkgconfig directly until we can bump various minimum requirements at pacemaker 2.1.0.

Reply:
Reasonable, or perhaps there's some kind of non-failing or
catchable way of expressing about the same.

Reply:
also ... or `pkgconf-0.8.10` (2013-02-24)

Reply:
Resolved now